weekly-mission/next-project/pages/api/fetchFolderLinksData.js

```diff
@@ -0,0 +1,15 @@
+export async function fetchLinkData(folderId) {
+  let query = "";
+  if (folderId) {
+    query = `?folderId=${folderId}`;
+  }
```

Comment:
```suggestion
  const query = folderId ? `?folderId=${folderId}` : '';
```

최대한 const 로 쓸 수 있는 것들에 대해서는 const 써주세요!